### PR TITLE
cdc: update output-raw-change-event comment link

### PR DESCRIPTION
**Pull request by 3AceShowHand**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.3 (TiDB 8.3 versions)
- [x] v8.2 (TiDB 8.2 versions)
- [x] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by CharlesCheung96 — APPROVED** [review]

**Comment by 3AceShowHand** [comment]
This is related to https://github.com/pingcap/docs-cn/pull/17699

**Review by flowbehappy — APPROVED** [review]

File: ticdc/ticdc-changefeed-config.md [hunk 1]
```diff
@@ -288,7 +288,7 @@ sasl-oauth-grant-type = "client_credentials"
 # Kafka SASL OAUTHBEARER 认证机制中的 audience。默认值为空。在使用该认证机制时，该参数可选填。
 sasl-oauth-audience = "kafka"
 
-# 控制是否输出原始的数据变更事件，默认值为 false，表示当使用非 MySQL Sink 且 `UPDATE` 事件的主键或者非空唯一索引的列值发生改变时，TiCDC 会将该其拆分为 `DELETE` 和 `INSERT` 两条事件，并确保所有事件按照 `DELETE` 事件在 `INSERT` 事件之前的顺序进行排序。设置为 true 时，表示不拆分事件，直接输出原始事件。
+# 控制是否输出原始的数据变更事件，默认值为 false。请参考 https://docs.pingcap.com/zh/tidb/dev/ticdc-split-update-behavior#控制是否拆分主键或唯一键-update-事件
```
Comment (Oreoxmt) [inline ticdc/ticdc-changefeed-config.md:291]:
```suggestion
# 控制是否输出原始的数据变更事件，默认值为 false。
# 更多信息，请参考 https://docs.pingcap.com/zh/tidb/dev/ticdc-split-update-behavior#控制是否拆分主键或唯一键-update-事件
```

File: ticdc/ticdc-changefeed-config.md [hunk 2]
```diff
@@ -342,7 +342,7 @@ batching-max-publish-delay=10
 # Pulsar Producer 发送消息的超时时间，默认 30 秒。
 send-timeout=30
 
-# 控制是否输出原始的数据变更事件，默认值为 false，表示当使用非 MySQL Sink 且 `UPDATE` 事件的主键或者非空唯一索引的列值发生改变时，TiCDC 会将该其拆分为 `DELETE` 和 `INSERT` 两条事件，并确保所有事件按照 `DELETE` 事件在 `INSERT` 事件之前的顺序进行排序。设置为 true 时，表示不拆分事件，直接输出原始事件。
+# 控制是否输出原始的数据变更事件，默认值为 false。请参考 https://docs.pingcap.com/zh/tidb/dev/ticdc-split-update-behavior#控制是否拆分主键或唯一键-update-事件
```
Comment (Oreoxmt) [inline ticdc/ticdc-changefeed-config.md:345]:
```suggestion
# 控制是否输出原始的数据变更事件，默认值为 false。
# 更多信息，请参考 https://docs.pingcap.com/zh/tidb/dev/ticdc-split-update-behavior#控制是否拆分主键或唯一键-update-事件
```

File: ticdc/ticdc-changefeed-config.md [hunk 3]
```diff
@@ -358,6 +358,6 @@ file-expiration-days = 0
 file-cleanup-cron-spec = "0 0 2 * * *"
 # 上传单个文件的并发数，默认值为 1，表示禁用并发。
 flush-concurrency = 1
-# 控制是否输出原始的数据变更事件，默认值为 false，表示当使用非 MySQL Sink 且 `UPDATE` 事件的主键或者非空唯一索引的列值发生改变时，TiCDC 会将该其拆分为 `DELETE` 和 `INSERT` 两条事件，并确保所有事件按照 `DELETE` 事件在 `INSERT` 事件之前的顺序进行排序。设置为 true 时，表示不拆分事件，直接输出原始事件。
+# 控制是否输出原始的数据变更事件，默认值为 false。请参考 https://docs.pingcap.com/zh/tidb/dev/ticdc-split-update-behavior#控制是否拆分主键或唯一键-update-事件
```
Comment (Oreoxmt) [inline ticdc/ticdc-changefeed-config.md:361]:
```suggestion
# 控制是否输出原始的数据变更事件，默认值为 false。
# 更多信息，请参考 https://docs.pingcap.com/zh/tidb/dev/ticdc-split-update-behavior#控制是否拆分主键或唯一键-update-事件
```

**Review by Oreoxmt — APPROVED** [review]

File: ticdc/ticdc-changefeed-config.md [hunk 4]
```diff
@@ -288,7 +288,7 @@ sasl-oauth-grant-type = "client_credentials"
 # Kafka SASL OAUTHBEARER 认证机制中的 audience。默认值为空。在使用该认证机制时，该参数可选填。
 sasl-oauth-audience = "kafka"
 
-# 控制是否输出原始的数据变更事件，默认值为 false，表示当使用非 MySQL Sink 且 `UPDATE` 事件的主键或者非空唯一索引的列值发生改变时，TiCDC 会将该其拆分为 `DELETE` 和 `INSERT` 两条事件，并确保所有事件按照 `DELETE` 事件在 `INSERT` 事件之前的顺序进行排序。设置为 true 时，表示不拆分事件，直接输出原始事件。
+# 控制是否输出原始的数据变更事件，默认值为 false。更多信息，请参考 https://docs.pingcap.com/zh/tidb/dev/ticdc-split-update-behavior#控制是否拆分主键或唯一键-update-事件
```
Comment (hfxsd) [inline ticdc/ticdc-changefeed-config.md:291]:
```suggestion
# 控制是否输出原始的数据变更事件，默认值为 false。更多信息，请参考[控制是否拆分主键或唯一键 UPDATE 事件](https://docs.pingcap.com/zh/tidb/dev/ticdc-split-update-behavior#控制是否拆分主键或唯一键-update-事件)
```

File: ticdc/ticdc-changefeed-config.md [hunk 5]
```diff
@@ -342,7 +342,7 @@ batching-max-publish-delay=10
 # Pulsar Producer 发送消息的超时时间，默认 30 秒。
 send-timeout=30
 
-# 控制是否输出原始的数据变更事件，默认值为 false，表示当使用非 MySQL Sink 且 `UPDATE` 事件的主键或者非空唯一索引的列值发生改变时，TiCDC 会将该其拆分为 `DELETE` 和 `INSERT` 两条事件，并确保所有事件按照 `DELETE` 事件在 `INSERT` 事件之前的顺序进行排序。设置为 true 时，表示不拆分事件，直接输出原始事件。
+# 控制是否输出原始的数据变更事件，默认值为 false。更多信息，请参考 https://docs.pingcap.com/zh/tidb/dev/ticdc-split-update-behavior#控制是否拆分主键或唯一键-update-事件
```
Comment (hfxsd) [inline ticdc/ticdc-changefeed-config.md:345]:
```suggestion
# 控制是否输出原始的数据变更事件，默认值为 false。更多信息，请参考[控制是否拆分主键或唯一键 UPDATE 事件](https://docs.pingcap.com/zh/tidb/dev/ticdc-split-update-behavior#控制是否拆分主键或唯一键-update-事件)。
```

File: ticdc/ticdc-changefeed-config.md [hunk 6]
```diff
@@ -358,6 +358,6 @@ file-expiration-days = 0
 file-cleanup-cron-spec = "0 0 2 * * *"
 # 上传单个文件的并发数，默认值为 1，表示禁用并发。
 flush-concurrency = 1
-# 控制是否输出原始的数据变更事件，默认值为 false，表示当使用非 MySQL Sink 且 `UPDATE` 事件的主键或者非空唯一索引的列值发生改变时，TiCDC 会将该其拆分为 `DELETE` 和 `INSERT` 两条事件，并确保所有事件按照 `DELETE` 事件在 `INSERT` 事件之前的顺序进行排序。设置为 true 时，表示不拆分事件，直接输出原始事件。
+# 控制是否输出原始的数据变更事件，默认值为 false。更多信息，请参考 https://docs.pingcap.com/zh/tidb/dev/ticdc-split-update-behavior#控制是否拆分主键或唯一键-update-事件
```
Comment (hfxsd) [inline ticdc/ticdc-changefeed-config.md:361]:
```suggestion
# 控制是否输出原始的数据变更事件，默认值为 false。更多信息，请参考[控制是否拆分主键或唯一键 UPDATE 事件](https://docs.pingcap.com/zh/tidb/dev/ticdc-split-update-behavior#控制是否拆分主键或唯一键-update-事件)。
```

**Review by hfxsd — APPROVED** [review]

**Pull request merged**